utils/swift_build_support/swift_build_support/products/wasmstdlib.py

```diff
@@ -206,3 +209,101 @@ def add_extra_cmake_options(self):
                                   '-Xcc;-mthread-model;-Xcc;posix;'
                                   '-Xcc;-pthread;-Xcc;-ftls-model=local-exec')
         self.cmake_options.define('SWIFT_ENABLE_WASI_THREADS:BOOL', 'TRUE')
+
+
+class WasmSwiftSDK(product.Product):
+    @classmethod
+    def product_source_name(cls):
+        return "swift-sdk-generator"
```

Comment:
It could make more sense to move the entire `WasmSwiftSDK` class in a separate file (like we do for almost all build products except `WASILibc` and `WasmLLVMRuntimeLibs`) -- is there any reason for which we put this in the same file as `WasmStdlib`?

Reply:
That makes sense to me 👍